### ISSUE TITLE
Predictions vs Observations analysis notebook

### ISSUE THREAD

**Issue by herbiebradley** [ai4er-cdt/flood_risk_shipston#27]
Notebook to analyse whether NFM interventions are effective.
CSV file is in #flood_data channel.

**Comment by herbiebradley** [comment]
Just sticking this here since it's a convenient place:

### Results for Shipston-only models
| Model | Validation NSE (2010-2016) |
|--------|--------------------------------|
| **Tuned Vanilla LSTM**** | **0.8175** |
| [Vanilla 1D Conv model](https://github.com/ai4er-cdt/flood_risk_shipston/blob/425731e381bbc6ad004aed1fb13863bae0026cdb/src/models/conv.py#L108) | 0.4309 |
| [WaveNet](https://arxiv.org/abs/1609.03499) | 0.6975 |
| [FilterNet](https://github.com/Mikata-Project/FilterNet) | 0.5978 |
| Autoregressive* WaveNet | 0.3359 |
| Autoregressive* FilterNet | 0.602 |
| Autoregressive* LSTM | 0.6925 |

*Autoregressive here refers to including the previous 365 days of discharge data as an additional feature.
**Hyperparameters: 10 layers, 100 hidden units, dropout probability of 0.2, 200 epochs of training.

Temperature and precipitation were the baseline features used in all models. The training set consisted of the data from 1986-2010, and validation set was 2010-2016.
